node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java

```diff
@@ -180,12 +181,12 @@ public void deconstruct() {
             osUpgradeActivatorInterval = zone.system().isCd() ? Duration.ofSeconds(30) : Duration.ofMinutes(5);
             periodicRedeployInterval = Duration.ofMinutes(30);
             provisionedExpiry = Duration.ofHours(4);
-            rebalancerInterval = Duration.ofMinutes(40);
+            rebalancerInterval = deploymentExistsOnAllConfigServers ? Duration.ofMinutes(120) : Duration.ofMinutes(40);
             redeployMaintainerInterval = Duration.ofMinutes(1);
             // Need to be long enough for deployment to be finished for all config model versions
             // Should be equal to timeout for deployments
             reservationExpiry = zone.system().isCd() ? Duration.ofMinutes(5) : Duration.ofMinutes(30);
-            scalingSuggestionsInterval = Duration.ofMinutes(31);
+            scalingSuggestionsInterval = deploymentExistsOnAllConfigServers ? Duration.ofMinutes(91) : Duration.ofMinutes(31);
```

Comment:
This can also be reduced back now, it does not deploy (assume it was increased because of periodic interval change?)

Reply:
Yes, thanks, fixed